.go-version

```diff
@@ -0,0 +1 @@
+1.20.0
```

Comment:
Where is this file being used? The commit says it is for the [setup-go](https://github.com/actions/setup-go) GH action, but there is no reference to this file in the `setup-go` documentation, and we are pointing to the `go.mod` file at the GH configuration.